careers/software-engineer.md

```diff
@@ -30,6 +43,19 @@ An engineer I is capable of taking on well-defined technical problems and managi
 
 **Collaboration:** They pair frequently. They are able to quickly recognize when they are stuck and ask for help. For more complicated tasks, they leverage the knowledge of their peers to plan and execute their work on time. They provide constructive feedback in code review and know when they need help constructing meaningful feedback.
 
+### Responsibilities
+
+* Write and debug code for small features or components.
+* Conduct software testing and quality assurance on your own code.
+* Create technical documentation for your code, often with guidance.
+* Participate in code review and offer constructive feedback.
+* Learn the codebase, development processes, and team tools.
+* Work on small, self-contained tasks with clear instructions.
+* Participate in team meetings and provide progress updates.
+* Adhere to team norms, coding standards and best practices.
+* Troubleshoot basic software issues and bugs.
```

Comment:
something around project leading. I don't think EN1s should be project leads, but should help project leads and observe so they feel prepared to lead one as EN2